### PR TITLE
getmnemonic: use get_credentials internally

### PR DESCRIPTION
**Pull request by LeoComandini**

### CODE REVIEW

**Comment by jgriffiths** [comment]
I think we either need to rename this to `getcredentials`, or add `getcredentials` and have this command return `['mnemonic']` from the credentials

File: green_cli/common.py [hunk 1]
```diff
@@ -231,15 +231,16 @@ def setpin(session, pin, device_id):
     return context.authenticator.setpin(session, pin, device_id)
 
 @green.command()
-@click.option('--password', type=str, default='')
+@click.option('--password', default="", expose_value=False, callback=details_json)
 @with_login
 @print_result
-def getmnemonic(session, password):
+@with_gdk_resolve
+def getmnemonic(session, details):
```
Comment (jgriffiths) [inline green_cli/common.py:238]: This will now return credentials, not the mnemonic, so should be renamed.

Comment (LeoComandini) [reply green_cli/common.py]: Renamed to `getcredentials`

**Comment by jgriffiths** [comment]
utack ef5144be93b726919b9a4a2217f07471a983b433

**Pull request merged**